CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## [Unreleased](https://github.com/jonathan-s/django-sockpuppet/tree/HEAD)
 
+- improved tests documentation
+- fixed camelcase/pascalcase tools for Reflex names
```

Comment:
Changelog is actually automatically generated so this is not needed :). But thanks for thinking of it.

Reply:
ah - I thought that - do you use a tool here? looks quite interesting...

Reply:
You can find the config of it here: https://github.com/jonathan-s/django-sockpuppet/blob/master/.github/workflows/changelog.yml